practice/validator/site.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+/**
+ * The Validator takes in the user's input as they type into any given text box
+ * in the form and uses kestroke events to visually que the user that their
+ * input is either valid or not, by filling the field with a yellow background
+ * while invalid.
+ */
+
+/**
+ * Adds the invalid class to the given field (distinguished by a class).
+ *
+ * The invalid class has a yellow background in the site.css file.
+ */
+function makeFieldYellow(inputclass) {
```

Comment:
This stands true for several functions, but don't you start with  your variable 'inputclass' being something like 'input.className', I can't find where it gets stripped off but it seems redundant to add it again here